### ISSUE TITLE
Actions fails with there are multiple locks files, even if path is set

### ISSUE THREAD

**Issue by sma** [withastro/action#30]
I think, the `Check lockfiles` step should respect the `path`. 

I've a repo that contains multiple `package-lock.json` files which causes this failure:

```
elif [ $(find "." -name "package-lock.json") ]; then 
/var/run/act/workflow/2-composite-0.sh: line 17: [: ./package-lock.json: binary operator expected
```

The `$()` variable contains multiple lines and this is something the `[` doesn't like.

Using `$(find ${{ inputs.path }} -name "package-lock.json")` should do the trick.

**Comment by natemoo-re** [comment]
I'm curious if https://github.com/withastro/action/pull/31 addresses this issue?

**Comment by natemoo-re** [comment]
I believe this has been addressed in [`v1.0.4`](https://github.com/withastro/action/releases/tag/v1.0.4) and is now supported on the `v1` tag, but please open a new issue if that's not the case.
